doc/source/dev/contributor/ubuntu_guide.rst

```diff
@@ -204,9 +204,23 @@ With conda
 
 #. Initialize git submodules: ``git submodule update --init``.
 
-#. Do an in-place build: enter ``python3 setup.py build_ext --inplace``. |br| This will compile the C, C++, and Fortran code that comes with SciPy. We installed ``python3`` with Anaconda. ``setup.py`` is a script in the root directory of SciPy, which is why you have to be in the SciPy root directory to call it. ``build_ext`` is a command defined in ``setup.py``, and ``--inplace`` is an option we'll use to ensure that the compiling happens in the SciPy directory you already have rather than the default location for Python packages. By building in-place, you avoid having to re-build SciPy before you can test changes to the Python code.
-
-#. Test the build: enter ``python3 runtests.py -v``. ``runtests.py`` is another script in the SciPy root directory. It runs a suite of tests that make sure SciPy is working as it should, and ``-v`` activates the ``--verbose`` option to show all the test output. If the tests are successful, you now have a working development build of SciPy! You could stop here, but you would only be able to use this development build when the Python working directory is the SciPy root directory.
+#. Build SciPy: enter ``python3 dev.py build`` |br|
```

Comment:
What is `|br|` and is it needed

Reply:
This is the line break, defined later in the file. Not needed really just a shortcut that was already there, I'm happy to rework and remove :)